kong/plugins/http-log/schema.lua

```diff
@@ -16,6 +17,15 @@ return {
           { retry_count = { type = "integer", default = 10 }, },
           { queue_size = { type = "integer", default = 1 }, },
           { flush_timeout = { type = "number", default = 2 }, },
-    }, }, },
+          { header_name = typedefs.header_name({ default = "Authorization"}), },
```

Comment:
Do we really need a default for this config? It seems to be too specific to be included.

Also, is a single header good enough? Is there any reason people may want to specify more than one header?

Reply:
"Authorization" is the common one, hence the default. It remains unused unless a value is also provided.

For now a single header should do, afaik there is only 1 service that needs it (at least users want supported).

Let me know if you want me to change aything anyway

Reply:
I guess there no immediate scenario where setting more than one statically defined headers are needed, so let's merge this for now.